game-app/game-core/src/main/java/org/triplea/util/Awt.java

```diff
@@ -0,0 +1,36 @@
+package org.triplea.util;
```

Comment:
`game-core/org/triplea/util` has the potential to be an arbitrary grab-bag of utilities used throughout anywhere in the codebase. With the split of sub-projects the hope is that we can put these tools outside of `game-core`. It is a goal that we try to avoid growing `game-core` and instead push the functionality in `game-core` into other sub-modules until eventually nothing is left in `game-core`.

With this said, we need to be mindful of a few considerations for a utility class:
1. Is there potential for this to be used in other parts of the code base? Can we use this utility class to consume other, existing code? (Notably, if we have an API with a default type, I think we have other places where we buffer images). If we have potential use in other parts of the code, then a `lib/` sub-project module would potentially be appropriate.
2. How would this utility class grow and would others expect to find this functionality, here? If we put in any utility method that involved `java.awt`, seemingly this would become unwieldly. An `ImageUtil` class, or `BufferedImageUtil` class name might be more fitting (but needs some thought and more context of where else we might be using this utility. Based on that we would have more information about which other methods might be desirable to add into this class).

Reply:
Interesting thoughts and questions. Only I cannot answer them. I happily restructure according to your advice. Knowing that you want to move from Awt to something more modern, I'd prefer to invest time in more future oriented considerations, e.g. how to make such code management thoughts as yours more available than as a PR comment.